manifests/init.pp

```diff
@@ -2,8 +2,20 @@
 #
 class lvm (
   $volume_groups = {},
+  $version       = 'installed'
 ) {
 
+  $real_provider = $::osfamily ? {
+    'Debian' => 'apt',
+    'RedHat' => 'yum'
+  }
+
+  package {
```

Comment:
Initially wasn't this patch only to install the package on deb systems? If so, the package resource should be wrapped in an `if $::osfamily == 'Debian'` block.